content/blog/2016/2016-01-10-beautiful-jenkins-dashboard.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "A beautiful Jenkins dashboard"
+author: kj187
+---
+
+
+Jenkins offers a handful of third party dashboards, but none of them is really beautiful and flexible from my point of view.
```

Comment:
I do that some lines below "... The advantage over an native Java Jenkins plugin is ..."